imap_processing/ultra/constants.py

```diff
@@ -129,3 +130,11 @@ class UltraConstants:
         341.989454569026,
         1e5,
     ]
+
+    ULTRA_FWHM_SCATTERING_CULLING_THRESHOLDS: ClassVar[dict] = {
```

Comment:
Might be nice to add a quick comment about what the indices are here for quick reference.

would this be better as a list that you could do a searchsorted on to find the bin?

```
CULLING_VALUES = np.array([-np.inf, 1, 5, 8, 10, 20])
CULLING_THRESHOLDS = np.array([np.nan, 12, 10, 8, 6, 4])
```